src/pages/infopages/OrderInformation.tsx

```diff
@@ -0,0 +1,49 @@
+import React, { FC } from 'react';
+import styled from 'styled-components';
+import { useTranslation, Trans } from 'react-i18next';
+import Heading from '../../components/Heading';
+import NormalText from '../../components/NormalText';
+import { Link as MuiLink } from '@material-ui/core';
+
+import Card from '../../components/Card';
+import { ContactInformation } from '../../utils/constants';
+
+const StyledInformationWrapper = styled.div`
+  width: 60%;
+  @media (max-width: ${({ theme }) => theme.breakpoints.values.sm + 'px'}) {
+    width: 90%;
+  }
+  padding-top: 4rem;
+  padding-bottom: 1rem;
+`;
+
+const StyledHeading = styled(Heading)`
+  text-align: center;
+  padding-bottom: 1rem;
+`;
+
+const OrderInformation: FC = () => {
+  const { t } = useTranslation('common');
```

Comment:
bruke `common` eller opprette ny oversettelses-fil? kommer to infopages til etterhvert

Reply:
sikkert like greit å bare lage egen fil med en gang? regner med det er tekst som kan være ganske lang også?

Reply:
ja, jeg heller mer og mer mot at vi prøver å bare ha generelle ting som kan brukes overalt i common

Reply:
ok, den er grei 👍